lib/command_runner.dart

```diff
@@ -49,17 +49,20 @@ class CommandRunner<T> {
 
   /// Returns [usage] with [description] removed from the beginning.
   String get _usageWithoutDescription {
+    const usagePrefix = "Usage:";
```

Comment:
Nit: In the args package, we don't bother with `const` or `final` for locals. Functions are small enough that it's not worth the effort.